middleware/server_header.go

```diff
@@ -39,6 +39,12 @@ func ServerHeader(name, version string) echo.MiddlewareFunc {
 				span.Description = helpers.CurrFuncName()
 				defer span.Finish()
 			}
+
+			if version == "" {
+				c.Response().Header().Set(echo.HeaderServer, name)
+				return next(c)
+			}
+
 			c.Response().Header().Set(echo.HeaderServer, name+"/"+version)
```

Comment:
I think this sentence can realize the logic
`c.Response().Header().Set(echo.HeaderServer, path.Join(name,version))`

Reply:
Thanks. your code is better. Fixed.